### ISSUE TITLE
[UI] Add compact mode

### ISSUE THREAD

**Issue by roxblnfk** [buggregator/frontend#68]
It would be great if you could add an option to activate a compact mode for the UI. In this mode, the left panel would be significantly reduced and the paddings would be trimmed.

I'm working on a small monitor and such a mode would help to fit more content in view.

**Comment by leonardocustodio** [comment]
I agree that a compact mode would be awesome, especially on the logging page. It seems there is so much "wasted" space between each log, we can see it fits way more logs on Ray.

![image](https://github.com/buggregator/frontend/assets/5619696/8a632f12-58fb-4707-87fb-dc0dc7d198b5)
![image](https://github.com/buggregator/frontend/assets/5619696/6f520470-e4e5-41a6-bfd5-11bc988ee8da)

**Comment by roxblnfk** [comment]
I think in this case it would be useful to use different layouts depending on the open tab.

For example, we don't need to display the `log` badge if we are already in the Logs tab. At the same time, we can optimize the logs page for more convenient viewing of logs (rather than dumps or others).
For example:
- display the time in the left short column
- display log level in the second column
- Display source (`ray`) and content type (`json`) in another place (mb before tags) and remove the top line with labels
- merge file with line
- use `white-space: nowrap; overflow: hidden;` for content
